client.go

```diff
@@ -0,0 +1,67 @@
+package litefs
+
+import (
+	"context"
+	"net/http"
+)
+
+// SubscribeEvents subscribes to events from the default (localhost:20202)
+// LiteFS node.
+func SubscribeEvents() EventSubscription {
+	return DefaultClient.SubscribeEvents()
+}
+
+// MonitorPrimary monitors the primary status of the LiteFS cluster via the
+// default (localhost:20202) LiteFS node's event stream.
+func MonitorPrimary() PrimaryMonitor {
+	return DefaultClient.MonitorPrimary()
+}
+
+// Client is an HTTP client for communicating with a LiteFS node.
+type Client struct {
+	// Base URL of the LiteFS cluster node.
+	URL string
+
+	// HTTP client to use for requests to cluster node.
+	HTTP *http.Client
+}
+
+// DefaultClient is a client for communicating with the default
+// (localhost:20202) LiteFS node.
+var DefaultClient = &Client{
+	URL:  "http://localhost:20202",
+	HTTP: http.DefaultClient,
+}
```

Comment:
This works. You could also have `NewClient()` return a client with defaults set.